packages/dx-react-grid/src/ts.test.js

```diff
@@ -0,0 +1,14 @@
+import { execSync } from 'child_process';
```

Comment:
Can we remove this test? Seems like we already testing typescript compilation in demos.